src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,152 @@
+- `description` - (Optional) Text that will be displayed above the form. Limited to 140 characters.
+
+## Example
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "alert-rule-action",
+      "title": "Create a Rekall Inc task",
+      "settings": {
+        "type": "alert-rule-settings",
+        "uri": "/sentry/alert-rule",
+        "required_fields": [
+          {
+            "type": "select",
+            "label": "Project",
+            "name": "project",
+            "options": [["1", "cornflakes"]]
+          },
+          {
+            "type": "select",
+            "label": "Assignee",
+            "name": "assignee",
+            "url": "/sentry/alert-rule/options/users/"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+
+## Issue Alert Request Format
+
+When an issue alert fires, your service will need to read the settings from the alert payload. The `settings` are in `data.issue_alert.settings`. Check out the full [Issue Alert webhook documentation](/product/integrations/integration-platform/webhooks/#issue-alerts) for more information.
+
+### Payload
+
+```json
+{
+  ...
+  "data": {
+    ...
+    "issue_alert": {
+      ...
+      "settings": [
+        {
+          "name": "title",
+          "value": "Ticket Title"
+        },
+        {
+          "name": "description",
+          "value": "Ticket Description"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Metric Alert Request Format
+
+When a metric alert fires, your service will need to read the settings from the alert payload. The `settings` in this example are nested in `data.metric_alert.alert_rule.triggers[1].actions[0].settings`. You will have to identify which trigger and action contains your settings based on the webhook type, so `triggers[1]` and `actions[0]` may not be accurate in your case. Check out the full [Metric Alert webhook documentation](/product/integrations/integration-platform/webhooks/#metric-alerts) for more information.
+
+### Payload
+
+```json
+{
+  “data”: {
+    ...
+    “metric_alert”: {
+      ...
+      “alert_rule”: {
+        ...
+        “triggers”: [
+          ...
+          {
+            ...
+            “actions”: [
+              {
+                ...
+                “settings”: [
+                  {
+                    “name”: “title”,
+                    “value”: “Critical Trigger”
+                  },
+                  {
+                    “name”: “description”,
+                    “value”: “Critical Description”
+                  }
+                ],
+
+              }
+            ]
+          },
+        ],
+
+      },
+    },
+  },
+}
+```
+
+### Alert Rule Creation Error Handling
+
+If a user setting up an alert rule action with your app tries to save an incorrect or malformed configuration, you can surface the errors to them directly in [sentry.io](https://sentry.io) by responding to the POST request with a JSON response in this shape:
+
+```json
+{message: string}
```

Comment:
JSON formatting nit again! 
- `message` should be in double quotes
- instead of 'string' you could put an example like "Channel no longer exists!"  or something, but if you want to keep 'string', I'd replace it with `<String>` like we have used above.